voter/index.md

```diff
@@ -476,7 +413,7 @@ Now, restart the application. Remember to use the `--reset-once` flag to recogni
 starport serve --reset-once
 ```
 
-Each time you reset the application state, all of the data from your previously created state is saved.
+Each time you reset the application state, all of the data from your previously created state is not saved.
```

Comment:
oof! I didn't know this. Not saving data is expected behavior?

Reply:
Default behavior is saving the data. When using `--reset-once` state though, the state is not saved.